salt/modules/consul.py

```diff
@@ -209,11 +225,10 @@ def get(consul_url=None, key=None, recurse=False, decode=False, raw=False):
                     item['Value'] = base64.b64decode(item['Value'])
                 else:
                     item['Value'] = ""
-
     return ret
 
 
-def put(consul_url=None, key=None, value=None, **kwargs):
+def put(consul_url=None, token=None, key=None, value=None, **kwargs):
```

Comment:
Same comment here.

There are quite a few functions below where the `token` option needs to be moved. Please update this anywhere else in this PR below as well.